packages/common/src/hooks/useFormattedAudioBalance.ts

```diff
@@ -25,7 +25,7 @@ export const useFormattedAudioBalance = (): UseFormattedAudioBalanceReturn => {
   const audioBalanceFormatted = formatWei(audioBalance, true, 0)
   const isAudioBalanceLoading = isNullOrUndefined(audioBalance)
 
-  const { data: audioPriceData, isPending: isAudioPriceLoading } =
+  const { data: audioPriceData, isLoading: isAudioPriceLoading } =
```

Comment:
so i thought pending = idle + loading, shouldn't that work as expected?

Reply:
I think `isLoading` is `isPending && isFetching` [inclusive](https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5/#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading:~:text=Lastly%2C%20a%20new%20derived%20isLoading%20flag%20has%20been%20added%20to%20the%20queries%20that%20is%20implemented%20as%20isPending%20%26%26%20isFetching)